playwright/tests/apps.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { AppsPage } from "@pages/appsPage";
+import { AppInstallationPage } from "@pages/appInstallationPage";
+import { expect, test } from "@playwright/test";
+import { AppPage } from "@pages/appPageThirdparty";
+import { APPS } from "@data/e2eTestData";
+
+test.use({ storageState: "playwright/.auth/admin.json" });
+let appsPage: AppsPage;
+let installationPage: AppInstallationPage;
+let appPage: AppPage;
+
+test("TC: SALEOR_119 User should be able to install and configure app from manifest @e2e", async ({
+  page,
+}) => {
+  const appsPage = new AppsPage(page);
+  const installationPage = new AppInstallationPage(page);
+
+  await appsPage.gotoAppsList();
+  await expect(appsPage.installExternalAppButton).toBeVisible();
+  await appsPage.installExternalAppButton.click();
+  await appsPage.typeManifestUrl("https://klaviyo.saleor.app/api/manifest");
```

Comment:
question: Why Klaviyo app specifically? Maybe we should have a debug app that will return data parsed from JWT and check if it's valid by using `app-sdk`

Reply:
@witoszekdev I chose an application that did not throw an error when entering a random value 😅

Reply:
We'll create a debug app just for that, that will validate JWT after clicking a button. Let's leave it as is for now